Tests/Homework3/1/student_tests.json

```diff
@@ -0,0 +1,44 @@
+{
+  "tests": [
```

Comment:
@yaniv-cohen Found some bugs I had thanks to this lol.

Fixed some broken tests, may you run again and check if it goes ok for your code?

Reply:
test handle neg K value, DRAW and invalid inputs  did not work for me
publishing correct test

Reply:
@yaniv-cohen What did you change?